cmd/daemon.go

```diff
@@ -2,19 +2,20 @@ package cmd
 
 import (
 	"fmt"
-	"github.com/textileio/go-textile/common"
-	"github.com/textileio/go-textile/core"
-	"github.com/textileio/go-textile/gateway"
-	"github.com/textileio/go-textile/pb"
-	"github.com/textileio/go-textile/repo"
-	"github.com/textileio/go-textile/util"
 	"net/http"
 	"os"
 	"os/signal"
 	"runtime"
 	"runtime/debug"
 	"strconv"
 	"time"
+
+	"github.com/textileio/go-textile/common"
```

Comment:
@balupton can you setup a `go imports` watcher in go land? This way me just opening a file doesn't cause changes.

Reply:
Should already be configured...

Reply:
Sure thing